source/CardGames.py

```diff
@@ -156,3 +156,27 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+
+def printHands(players: dict):
+    # Show each player's hand
+    for player in players.values():
+        print(f'{player.name}:')
+        player.showHand(True)
+        print()
+
+def player_ages(players):
+    playerAge = []
+    for player in players.values():
+        while True:
+            try:
+                age = int(input("Enter {}'s age: ".format(player.name)))
+                playerAge.append((player, age))
+                break
+            except ValueError:
```

Comment:
Great use of the except value; it improves the codes robustness, code looks good, great job